src/tools/dev/scripts/bv_support/bv_vtk.sh

```diff
@@ -1278,54 +1533,79 @@ EOF
 
 function apply_vtk_patch
 {
-    apply_vtk9_gcc13_patch
-    if [[ $? != 0 ]] ; then
-        return 1
-    fi
-    apply_vtk9_allow_onscreen_and_osmesa_patch
-    if [[ $? != 0 ]] ; then
-        return 1
-    fi
+    if test "${VTK_VERSION}" = "9.2.6" ; then
```

Comment:
I personally think `if [[ "${VTK_VERSION}" == "9.2.6" ]];` is more idiomatic than using `test`.

Reply:
done.